tests/data_multi/config_file.json

```diff
@@ -0,0 +1,64 @@
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/number_of_cycles/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/detector_type": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/FABRICATION[device_information]/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/FABRICATION[device_information]/serial_number": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/detector_number": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/detector_TYPE[detector_ccd]/gain_setting": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/type": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/power": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SOURCE[source_532nmlaser]/power/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/FABRICATION[device_information]/vendor": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/FABRICATION[device_information]/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/type": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/FABRICATION[device_information]/vendor": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/FABRICATION[device_information]/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/magnification": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/LENS_OPT[objective_lens]/numerical_aperture": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/device_information/model": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/grating/period": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/grating/period/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/wavelength": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/MONOCHROMATOR[monochromator]/wavelength/@units": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/attached_to": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/measurement": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/value": "@eln",
+  "/ENTRY[entry]/INSTRUMENT[instrument]/SENSOR[temperature_sensor]/value/@units": "@eln",
+  "/ENTRY[entry]/SAMPLE[sample]/name": "@eln",
+  "/ENTRY[entry]/SAMPLE[sample]/atom_types": "@eln",
+  "/ENTRY[entry]/USER[user]/address": "@eln",
+  "/ENTRY[entry]/USER[user]/affiliation": "@eln",
+  "/ENTRY[entry]/USER[user]/email": "@eln",
+  "/ENTRY[entry]/USER[user]/name": "@eln",
+  "/ENTRY[entry]/experiment_description": "@eln",
+  "/ENTRY[entry]/experiment_type": "@eln",
+  "/ENTRY[entry]/raman_experiment_type": "@eln",
+  "/ENTRY[entry]/title": "@eln",
+  "/ENTRY[entry]/start_time": "@eln",
+  "/ENTRY[entry]/definition": "@eln",
+  "/ENTRY[entry]/definition/@url": "@eln",

```

Comment:
These two can be removed as they are auto-filled by the dataconverter. That's probably one of the reasons why the url attribute makes problems.

Reply:
That is right, it is filled automatically.
But the error still occours, as NXraman requires the URL (which is not the case for NXxps, NXem or NXstm, etc.).
This is the reason why this was not found before.

For sake of time efficiency, I would keep it for now and try to take a look, how this can be handled in pynxtools later correctly.

Reply:
You can simpy remove it here, this is independent of the uppercase `@URL` problem.

Reply:
I removed both entries. Nevertheless, pytest fails then. I got it. I wont test it a thrid time.

```
WARNING  pynxtools:helpers.py:124 Missing attribute: "/ENTRY[entry]/definition/@URL"
```

Reply:
Ok, then keep it in for now. We have the issues in pynxtools anyway.